pallets/msa/src/lib.rs

```diff
@@ -78,7 +78,6 @@ use common_primitives::{
 	msa::{
 		Delegation, DelegationValidator, DelegatorId, MsaLookup, MsaValidator, ProviderId,
 		ProviderLookup, ProviderRegistryEntry, SchemaGrantValidator,
-		EXPECTED_MAX_NUMBER_OF_PROVIDERS_PER_DELEGATOR,
```

Comment:
We should delete this in common primitives as well.

Reply:
Fixed in next commit.